packages/sdk/src/types/runtime/actor.ts

```diff
@@ -27,6 +27,7 @@ import {
     SetAnimationStateOptions
 } from '../..';
 import { log } from '../../log';
+import { ZeroGuid } from '../../utils/deterministicGuids';

```

Comment:
>ZeroGuid [](start = 9, length = 8)

constants.ts would be a better place for this.

Reply:
Oh right, I forgot about that file.